unitylibs/core/workflow/workflow-photoshop/action-binder.js

```diff
@@ -24,6 +24,26 @@ export default class ActionBinder {
     this.errorToastEl = null;
     this.psApiConfig = this.getPsApiConfig();
     this.serviceHandler = null;
+    this.loadServiceHandler();
+    this.loadSVGsMaybe();
```

Comment:
This will add both progress circle and all the svgs (Including the ones that are present on the buttons that will be made visible only when user interacts) in the critical rendering path. Can we split this into two functions and call the progress circle js and css loader here and load svg in psActions?

Reply:
Maybe I'm missing something, but this will not work because we'll still load the other SVGs when the user clicks on "Remove background". 
This will still cause a delay between the button click and the `'show'` action for the progress circle.